### DISCUSSION TITLE
Please encourage manufacturers to make an ELRS-based USB dongle for wireless use with simulators

### DISCUSSION THREAD

**Discussion by GizmoChicken** [ExpressLRS/ExpressLRS#1784] [Ideas]
I successfully set up ELRS for use as a [Bluetooth joystick with simulators](https://www.youtube.com/watch?v=ILYE-JaXjOs).  It’s a great feature, and I very much appreciate that the ELRS devs included this feature as an option. Thanks to devs who are working on this!

Although I greatly appreciate having the ELRS Bluetooth joystick feature as an option, if I'm being honest, I still much prefer (and probably always will prefer) using my [FrSky XSR-SIM](https://www.frsky-rc.com/product/xsr-sim/) wireless USB dongle with simulators.  Simply put, the FrSky XSR-SIM offers a better experience compared to using the ELRS Bluetooth joystick feature.

What I'd prefer even more:   an **ELRS-based USB dongle for wireless use with simulators**.   Then, I could finally be done with D16!

The above in mind, I hope that, _with some encouragement from the ELRS community_, some brave manufacturer can be convinced to make an ELRS-based USB dongle for use with simulators.  Ideally, such a USB dongle would use an ELRS receiver that could be flashed directly using the ExpressLRS Configurator (that is, not SPI-based).

I've written to a few manufacturers, but without much response.  But if more who want this feature provide encouragement, maybe at least one brave manufacturer will take on the risk of bringing such a product to market.

I’ll just add I plugged an ELRS flight controller into my computer for use with a simulator, much as described Oscar Liang [here]( https://oscarliang.com/betaflight-fc-fpv-simulator/).  The approach works reasonably well, but devoting a flight controller is a bit overkill/wasteful.  Also, I’ve considered building an ELRS / CRSF HID Joystick based on a Pi Pico as described [here](https://www.youtube.com/watch?v=MCOWKvFTHRc) and [here](https://www.youtube.com/watch?v=Zuhfbobgm5A), but I’m too lazy.  :)

**Comment by fa-lk** [comment]
I searched for the same as my external module has very low Bluetooth range due to a massive heat sink. 

Seems like "HGLRC C1 dongle" and "ELRS Squid Stick" seem to exist in the meantime (I own non of them yet)

**Comment by ScottLangham** [comment]
One is available here: [https://elrsquidstick.com/](https://elrsquidstick.com/)

**Comment by VanLaser** [comment]
Couple of  DIY links, the first one is pretty hard to find on searches, for some reason:

- https://github.com/kkbin505/Simple_RX (Pro Micro + ELRS receiver, cheapest)  
- https://github.com/pfeerick/USB_ELRS_Receiver (Xiao RP2040 / SAMD21 + ELRS receiver)
